pom.xml

```diff
@@ -73,6 +73,17 @@
             <artifactId>mysql-connector-java</artifactId>
             <version>8.0.12</version>
         </dependency>
-    </dependencies>
 
+        <dependency>
+            <groupId>javax.el</groupId>
+            <artifactId>javax.el-api</artifactId>
+            <version>3.0.0</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.glassfish.web</groupId>
```

Comment:
I'm not sure, whether we really need this dependency. Add a comment below this comment, where do we use this dependency.